constellations/orion/src/domains/people.go

```diff
@@ -0,0 +1,82 @@
+package domains
+
+import (
+	"database/sql"
+	"errors"
+	"regexp"
+	"time"
+)
+
+var TABLE_PEOPLE = "people"
+
+//rowid,first,middle,last names, phone, emails, isguardian, guardianid, familyid
+type People struct {
+	Id         uint         `json:"id"`
+	CreatedAt  time.Time    `json:"-" db:"created_at"`
+	UpdatedAt  time.Time    `json:"-" db:"updated_at"`
+	DeletedAt  sql.NullTime `json:"-" db:"deleted_at"`
+	FirstName  string       `json:"firstName" db:"first_name"`
+	LastName   string       `json:"lastName" db:"last_name"`
+	MiddleName NullString   `json:"middleName" db:"middle_name"`
+	Email      string       `json:"email"`
+	Phone      string       `json:"phone"`
+	IsGuardian bool         `json:"isGuardian" db:"is_guardian"`
+	GuardianId NullUint     `json:"guardianId" db:"guardian_id"`
```

Comment:
I realize we don't need guardianId anymore. It actually restricts us a lot.

We can support multiple guardians to a family if we just have "familyId" and "isGuardian"
So we can have 6 users with the same familyId, 2 of them could be Guardians because isGuardian is true for both of them. The Primary email will be dependent on the user managing the family account.